geomesa-convert/geomesa-convert-redis-cache/src/main/scala/org/locationtech/geomesa/convert/redis/RedisEnrichmentCache.scala

```diff
@@ -64,14 +66,13 @@ class RedisEnrichmentCacheFactory extends EnrichmentCacheFactory {
   override def build(conf: Config): EnrichmentCache = {
     val Array(host, port) = parseRedisURL(conf.getString("redis-url"))
     val timeout = if (conf.hasPath("expiration")) conf.getLong("expiration") else -1
-    val connBuilder = new RedisConnectionBuilder {
-      val pool = new JedisPool(host, port.toInt)
-      override def getConn: Jedis = {
-        pool.getResource
-      }
+    val connBuilder: RedisConnectionBuilder = new RedisConnectionBuilder {
```

Comment:
Since this is a pool are connections returned?

Reply:
no, they are closed up on line 52, which is how you 'return' them